### PR TITLE
Updates MIT brand and fixes tooling

### PR DESCRIPTION
**Pull request by JPrevost**
Updates our style guide to use an updated MIT logo, which includes using a CDN for 4 SVGs rather than embedding them in this repository.

Updates tooling for this repo to work again (by removing gulp). Some of it is missing, but since it didn't work anymore anyway getting SCSS compilation working again and documenting the remaining shortcomings felt okay for now.

### Relevant ticket(s)

* https://mitlibraries.atlassian.net/browse/PW-80

### CODE REVIEW

**Comment by JPrevost** [comment]
Note: there is a chance this doesn't build properly in GitHub Pages once merged. Jekyll runs properly locally, but there was a note that something I removed was originally added to help this run properly. If it breaks on merge, I'll fix it... but I didn't want to leave the extra complexity as there is a chance it was no longer needed. 🤞🏻

File: _assets/sass/elements/_footer.scss [hunk 1]
```diff
@@ -7,7 +7,8 @@
 }
 
 .wrap-footer {
-    background: $black url('#{$image-path}/vi-shape7-tp.svg') no-repeat 10% center;
+    background-image: url('https://cdn.mitlibrary.net/files/branding/local/vi-shape7-tp.svg');
```
Comment (jazairi) [inline _assets/sass/elements/_footer.scss:10]: Are we intentionally using the stage URLs?

File: _includes/footer-site-slim.html [hunk 2]
```diff
@@ -3,79 +3,59 @@
...
-            </div><!-- end .identity -->
+      <div class="footer-main" aria-label="MIT Libraries footer">
+        <div class="identity">
+          <div class="wrap-logo-lib">
+            <a href="https://libraries.mit.edu" class="logo-mit-lib" alt="MIT Libraries Logo">
+              <span class="sr">MIT Libraries home</span>
+              <img
+                src="https://cdn.dev1.mitlibrary.net/files/branding/local/mitlib-wordmark.svg"
+                alt="MIT Libraries logo"
+                height="38"
```
Comment (jazairi) [inline _includes/footer-site-slim.html:16]: I believe we'd agreed on 35px. Ultimately, I don't really care, but I want to make it's consistent. I asked in #engx what everyone is using to confirm.

File: _includes/footer-site.html [hunk 3]
```diff
@@ -109,13 +87,36 @@ <h3 class="title item"><a href="https://libraries.mit.edu/about">About us</a></h
     <footer class="footer-info-institute" aria-label="MIT footer" role="contentinfo">
       <a class="link-logo-mit" href="https://www.mit.edu">
         <span class="sr">MIT</span>
-        <svg x="0" y="0" width="54" height="28" viewBox="0 0 54 28" enable-background="new 0 0 54 28" xml:space="preserve" class="logo-mit"><rect x="28.9" y="8.9" width="5.8" height="19.1" class="color"/><rect width="5.8" height="28"/><rect x="9.6" width="5.8" height="18.8"/><rect x="19.3" width="5.8" height="28"/><rect x="38.5" y="8.9" width="5.8" height="19.1"/><rect x="38.8" width="15.2" height="5.6"/><rect x="28.9" width="5.8" height="5.6"/></svg>
+        <svg
```
Comment (jazairi) [inline _includes/footer-site.html:90]: We should use the CDN SVG here.

**Review by jazairi — CHANGES_REQUESTED** [review]
Thanks for this long-overdue maintenance. I'm not at all worried about manual editing of JS, since we use so little of it (and, as you mention in the readme, it's unlikely to change often).

Some of the links to the CDN use the stage or dev site; I think these should all be prod unless there's a good reason for them not to. (I mentioned one in a comment, but there are a few others.) There's also one SVG element that I think can come from the CDN.

We also need to confirm the height of all logos. We'd said 35px, but this uses 38px, and Matt is doing 32px for WP, so there's probably more discussion to have there.

File: _assets/sass/elements/_footer.scss (hunk 1 above)
Comment (JPrevost) [reply _assets/sass/elements/_footer.scss:10]: No, but that is a link to prod...I suspect there are others I missed and with check them more closely :)

Comment (jazairi) [reply _assets/sass/elements/_footer.scss:10]: `https://cdn.libraries.mit.edu` is an alias, then? TIL...

**Review by jazairi — APPROVED** [review]

File: _assets/sass/elements/_footer.scss (hunk 1 above)
Comment (JPrevost) [reply _assets/sass/elements/_footer.scss:10]: Oh... what were you using for prod? I didn't realize there was a different link that worked for prod...  Interesting. But yeah, the non-environment link is probably best to use in production.

Comment (jazairi) [reply _assets/sass/elements/_footer.scss:10]: Yeah, `https://cdn.libraries.mit.edu` is what I'm using for Springshare. It seems to work. 🤷

**Pull request merged**